### DISCUSSION TITLE
adding "overflow" argument to Console

### DISCUSSION THREAD

**Discussion by dlax** [Textualize/rich#1803] [Ideas]
I want to use `Live` with content that would overflow horizontally and would thus like to control this similarly to how it works with `console.print()`. Unless I missed it, there's no way to control calls to `console.print()` emitted by the `Live` instance upon refresh. But since it is possible to pass a custom console to the Live object, this use case could be solved that way.

That's my use case now, but it seems to me that controlling overflow globally (similarly to how, e.g., soft-wrapping can currently be controlled) would make sense in general.

So, can we add an `overflow` argument to `Console.__init__()`?

If so, I'd be happy to work on a PR.

**Comment by willmcgugan** [comment]
Can you give me more details about what you are trying to achieve here? Maybe paste some code?

**Reply by dlax** [reply]
Hm.... While trying to build a minimal example, I realize that everything works as expected already. So there was something wrong with the complex case I was trying to handle.
Sorry for the noise and thank you for this great library!
